tests/config.rs

```diff
@@ -760,11 +760,7 @@ async fn parses_sink_full_es_basic_auth() {
     .unwrap();
 }
 
-#[cfg(all(
-    feature = "docker",
```

Comment:
Not sure why this feature constraint was here as the feature didn't exist, but it flagged this test as failing to compile when I reintroduced it.